### ISSUE TITLE
Add a button to "autofill" Refresh Token

### ISSUE THREAD

**Issue by potmat** [potmat/homebridge-google-nest-sdm#16]
The process of getting the refresh token is difficult and error prone, especially for people who are not computer savvy (e.g. using a "curl" command).  It should be possible to have a button in the config UI that will open the authentication screen and get the value.

**Comment by krehman** [comment]
Can you not potentially use the Terminal in Homebridge UI?

**Comment by Daxulan** [comment]
yes i'm blocked at this point , i have all my code but curl is a mystery for me and i try but don't work ..

**Comment by krehman** [comment]
I did this process a while back so don't really recall off the top of my head, but anywhere in the instructions, e.g. [Use the API](https://developers.google.com/nest/device-access/use-the-api), it asked to use the `curl` command, I just opened up the **Terminal** from the _Homebridge-UI_ under the menu and inputted the command.

![image](https://user-images.githubusercontent.com/6234445/171038438-6da20f1f-82b8-4785-ae89-34e5ae4a805d.png)

**Comment by litex2x** [comment]
I don't even think the refresh token works properly? I keep having to get a new one every week.
